public-api/config/cdr_versions_test.json

```diff
@@ -11,5 +11,18 @@
     "numParticipants": 946237,
     "cdrDbName": "cdr20181023",
     "publicDbName": "public20181015"
+  },
+  {
```

Comment:
Please just replace the other one - it never existed (you can just use the same ID)

Reply:
Also please update `cdr_versions_local.json`

Reply:
Actually nvm